.github/workflows/release-please.yml

```diff
@@ -0,0 +1,71 @@
+name: Run Release Please
+
+on:
+  push:
+    branches: [ main ]
+
+jobs:
+  release-package:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write # Needed if using OIDC to get release secrets.
+      contents: write # Contents and pull-requests are for release-please to make releases.
+      pull-requests: write
+    outputs:
+      release-created: ${{ steps.release.outputs.release_created }}
+      upload-tag-name: ${{ steps.release.outputs.tag_name }}
+      package-hashes: ${{ steps.build.outputs.package-hashes}}
+    steps:
+      - uses: google-github-actions/release-please-action@v3
```

Comment:
Would using @v4 work?

Reply:
Maybe. Ryan had a similar question on the ruby otel branch. I'm just keeping my repos all using the same version of release please for now, and then I'll go through and update them in mass. This way it's a one and done thing. It also let's this be a copy-paste job, so I'm confident the first release will work.